examples/cross_val/scripts/cross_val_algo_random_forest.py

```diff
@@ -16,31 +16,63 @@
 import os
 import json
 
+import numpy as np
+from sklearn.model_selection import KFold
+
 import substra
 
 current_directory = os.path.dirname(__file__)
-assets_keys_path = os.path.join(current_directory, '../../titanic/assets_keys.json')
 folds_keys_path = os.path.join(current_directory, '../folds_keys.json')
 
-with open(os.path.join(current_directory, '../../config.json'), 'r') as f:
+N_FOLDS = 2 
+NODE_ID = 1
+ALGO_NAME = "Random Forest"
+DATASET_NAME = "Titanic"
+OBJECTIVE_NAME = "Titanic"
+
+with open(os.path.join(current_directory, f'../../config_node{NODE_ID}.json'), 'r') as f:
     config = json.load(f)
 
 client = substra.Client()
 client.add_profile(config['profile_name'], config['username'], config['password'],  config['url'])
 client.login()
 
-print(f'Loading existing asset keys from {os.path.abspath(assets_keys_path)}...')
-with open(assets_keys_path, 'r') as f:
-    assets_keys = json.load(f)
+# -----------------------------------
+# Retrieve objective and dataset keys
+# -----------------------------------
+print('Retrieving existing asset keys ...')
+algo_key = [d['key'] for d in client.list_algo() if ALGO_NAME in d['name']][0]
+data_manager_key = [d['key'] for d in client.list_dataset() if DATASET_NAME in
+                    d['name']][0]
+objective_key = [d['key'] for d in client.list_objective() if OBJECTIVE_NAME in 
+                 d['name']][0]
+train_data_sample_keys = client.get_dataset(data_manager_key)['trainDataSampleKeys']
 
-algo_key = assets_keys['algo_random_forest']['algo_key']
-objective_key = assets_keys['objective_key']
-dataset_key = assets_keys['dataset_key']
+# -----------------------------------
+# Retrieve objective and dataset keys
+# -----------------------------------
+print('Generating folds...')
+X = np.array(train_data_sample_keys)
+kf = KFold(n_splits=N_FOLDS, shuffle=True)
+folds_keys = {
+    'folds': [
+        {
+            'train_data_sample_keys': list(X[train_index]),
+            'test_data_sample_keys': list(X[test_index])
+        } for train_index, test_index in kf.split(X)
+    ]
+}
+
+# saving folds keys
+with open(os.path.join(current_directory, folds_keys_path), 'w') as f:
+    json.dump(folds_keys, f, indent=2)
```

Comment:
The scripts fails with the following error when run from the cross_val directory:
```
Traceback (most recent call last):
  File "scripts/cross_val_algo_random_forest.py", line 67, in <module>
    with open(os.path.join(current_directory, folds_keys_path), 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'scripts/scripts/../folds_keys.json'
```
The file name should probably just be `folds_keys_path`.